src/plugin.js

```diff
@@ -815,12 +815,16 @@ for (let i = 0; i < _readOnly.length; i++) {
  * Check if the Flash tech is currently supported.
  *
  * @return {boolean}
- *          - True if the flash tech is supported.
- *          - False otherwise.
+ *          - True for Chrome and Safari Desktop and if flash tech is supported
+ *          - False otherwise
  */
 Flash.isSupported = function() {
+  // for Chrome Desktop and Safari Desktop
+  if (videojs.browser.IS_CHROME || videojs.browser.IS_SAFARI) {
```

Comment:
I think we need to make this a bit stricter:
```js
if (videojs.browser.IS_CHROME && !videojs.browser.IS_ANDRIOD ||
    videojs.browser.IS_SAFARI && !videojs.browser.IS_IOS) {
// ...
}
```
otherwise it'll return true on chrome for android and iOS.